groups/staging-windows-virtual-machines/proxy_test_ec2.tf

```diff
@@ -0,0 +1,85 @@
+# ------------------------------------------------------------------------------
+# Proxy Test Security Group and rules
+# ------------------------------------------------------------------------------
+module "proxy_test_ec2_security_group" {
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "~> 3.0"
+
+  name        = "sgr-${var.application}-proxy-test-server"
+  description = "Security group for the ${var.application} Proxy Test Server"
+  vpc_id      = data.aws_vpc.vpc.id
+
+  egress_rules = ["all-all"]
+}
+# ------------------------------------------------------------------------------
+# Proxy Test CloudWatch
+# ------------------------------------------------------------------------------
+
+resource "aws_cloudwatch_log_group" "proxy_test" {
+  for_each = local.proxy_test_cw_logs
+
+  name              = each.value["log_group_name"]
+  retention_in_days = lookup(each.value, "log_group_retention", var.default_log_group_retention_in_days)
+  kms_key_id        = lookup(each.value, "kms_key_id", data.aws_kms_key.logs.arn)
+
+  tags = merge(
+    local.default_tags,
+    map(
+      "Name", "${var.application}-proxy-test-server",
+      "ServiceTeam", var.ServiceTeam
+    )
+  )
+}
+
```

Comment:
I wouldnt build these, not necessary for a test

Reply:
removed